bridge/shared/interfaces/sequence-execution-result.ts

```diff
@@ -56,15 +51,12 @@ export interface SequenceExecutionStatus {
   previousTasks: [
     {
       name: string;
-      properties: {
-        // eslint-disable-next-line @typescript-eslint/ban-types
-        additionalProp1: {};
-      };
-      result: string;
-      status: string;
+      properties: Record<string, unknown>;
+      result: ResultTypes;
+      status: StatusTypes;
       triggeredID: string;
     }
   ];
-  state: string;
+  state: 'triggered' | 'waiting' | 'suspended' | 'paused' | 'finished' | 'cancelled' | 'timedOut';
```

Comment:
Could you also add 'started' here? And maybe move it into a separate type variable and use that for the status property of the sequence-execution API call in the apiService.